tests/ops/einsum/test_adjoint.py

```diff
@@ -117,7 +117,7 @@ def test_marginal(equation):
         assert_equal(expected, actual)
 
 
-@pytest.mark.filterwarnings("ignore:.*reduce_op is deprecated")
+@pytest.mark.filterwarnings("ignore:.*reduce_op` is deprecated")
```

Comment:
Can we replace the "`" with a ".?" or something so as to cover both versions of the deprecation warning?

Reply:
Yes, I've changed to "ignore:.*reduce_op`? is deprecated" which matches both versions.